Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -353,7 +353,14 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
                                                 callback:(FIRVerificationResultCallback)callback {
   [self verifyClientWithCompletion:^(FIRAuthAppCredential *_Nullable appCredential,
                                      NSError *_Nullable error) {
-    if (error) {
+    NSError *_Nullable underlyingError = error.userInfo[NSUnderlyingErrorKey];
+    BOOL isErrorSimulatorError = error.code == FIRAuthErrorCodeMissingAppToken
```

Comment:
How about wrapping in ifdef simulator code, so it only gets built for simulator libraries

Reply:
That should make it only available within simulator slices of fat binaries, right?

Reply:
Yep, that's another way of saying what I said :)

Reply:
Done.